Project.toml

```diff
@@ -23,9 +23,9 @@ Tables = "bd369af6-aec1-5ad0-b16a-f7cc5008161c"
 Unicode = "4ec0a83e-493e-50e2-b9ac-8f72acf5a8f5"
 
 [compat]
-CategoricalArrays = "0.9.3"
+CategoricalArrays = "0.10.0"
```

Comment:
How about allowing both since we support both? That will help ensuring that people don't use old DataFrames versions due to conflicts.

Reply:
I assumed that this will not be conflicting as we use CategoricalArrays.jl only in `[extras]` so it would be used only when testing the package.

@KristofferC - could you please confirm this is correct.

Also I assume soon we will have a new Julia LTS and then we will drop 1.0 support, when the `[extras]` section will be gone.

Reply:
Indeed. I've just checked, it's perfectly fine to install CategoricalArrays 0.8 and DataFrames 1.1 in the same environment.